webapp/src/pages/CreateChatRoomPage/CreateChatRoomPage.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  Button,
+  ButtonVariant,
+  Header,
+  IconVariant,
+  Input,
+  InputVariant,
+  MediumAvatar,
+  Text,
+  TextColor,
+  TextVariant,
+  TextWeight,
+} from '../../shared/components';
+import { CHAT_URL, WILDCARD_AVATAR_URL } from '../../shared/urls';
+import { useNavigate } from 'react-router-dom';
+import { useNavigation } from '../../shared/hooks/UseNavigation';
+import { useEffect, useState } from 'react';
+import { chatsApi } from '../../shared/services/ApiService';
+import { CreateChatDto, ResponseError } from '../../shared/generated';
+import './CreateChatRoomPage.css';
+
+type FormStatus = {
+  type: 'success' | 'error' | 'pending';
+  message: string;
+};
+
+const initialSubmitFormStatus: FormStatus = {
+  type: 'pending',
+  message: '',
+};
+
+export default function CreateChatRoomPage() {
```

Comment:
praise: I like how you structure this page and it works perfectly with the back-end :+1: